bundles/org.eclipse.orion.client.ui/web/orion/commandRegistry.js

```diff
@@ -456,7 +467,7 @@ define([
 							parent.setAttribute("role", "dialog"); //$NON-NLS-1$ //$NON-NLS-0$
 							parent.setAttribute("aria-modal", "true"); //$NON-NLS-1$ //$NON-NLS-0$
 							if (commandInvocation.domNode && commandInvocation.domNode.textContent) {
```

Comment:
Do we need this test still?

if (commandInvocation.domNode && commandInvocation.domNode.textContent) {

Reply:
Yes, we do :-)